phpBB/phpbb/template/twig/environment.php

```diff
@@ -77,7 +76,7 @@ public function __construct(assets_bag $assets_bag, config $phpbb_config, filesy
 
 		$this->phpbb_root_path = $this->phpbb_path_helper->get_phpbb_root_path();
 
-		$this->assets_bag = $assets_bag;
```

Comment:
Why not continuing using di?

Reply:
Well I only added the change before due to iconify so just undid it. But yes, we could also continue using DI for this.